policyengine_us/parameters/gov/states/ut/tax/income/credits/nonrefundable/eitc.yaml

```diff
@@ -0,0 +1,9 @@
+description: Utah matches this percent of the federal EITC.
+values:
+  2022-01-01: 0.15
```

Comment:
isn't it 20% ?

![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/6bca9cdc-7afb-4c1d-b041-6ac481b07102)

Reply:
I think the 15% comes from here: https://incometax.utah.gov/credits/earned-income-tax-credit#:~:text=You%20may%20claim%20a%20Utah,in%20the%20same%20tax%20year. Should we use 20% here?

Reply:
what does the tax form say? Maybe it is 20% in 2023

Reply:
It says 15% in the instructions for the 2022 tax form: https://tax.utah.gov/forms/current/tc-40inst.pdf#page=22

Reply:
You are absolutely right, can you add this source as well showing the 15% in 2022 https://law.justia.com/codes/utah/2022/title-59/chapter-10/part-10/section-1044/

Also add a 2023-01-01: 0.2 and a comment over the initial legal source saying that it increased in 2023 to 20%